### PR TITLE
Updating lab to use dbjson

### PR DESCRIPTION
**Pull request by Matt-Eva**

### CODE REVIEW

File: src/__tests__/Home.test.js [hunk 1]
```diff
@@ -12,17 +12,17 @@ test("renders 'Home Page' inside of an <h1 />", () => {
...
   expect(titleList[0].textContent).toBe("Doctor Strange");
 })
 
-test("Displays links for each associated movie", () =>{
+test("Displays links for each associated movie", async () =>{
   render(<RouterProvider router={router}/>);
-  const linkList = screen.queryAllByText(/View Info/);
+  const linkList = await screen.findAllByText(/View Info/);
   expect(linkList.length).toBeGreaterThan(2);
   expect(linkList[0].href).toBe("http://localhost/movie/Doctor%20Strange");
```
Comment (lizbur10) [inline src/__tests__/Home.test.js:27]:
```suggestion
  expect(linkList[0].href).toBe("http://localhost/movie/1");
```

**Review by lizbur10 — CHANGES_REQUESTED** [review]
Hey! I found a couple of issues:

1. The solution code for Actors.js and Directors.js both use a Card component. The component is present in the solution code, but there's nothing in the instructions about creating that component. (It's also confusingly named, imo.)
2. The tests for Actors and Directors both rely on the data.js file. I think that's fine for now - I don't think it's worth spending time on at this point - but maybe change the file name to something more explicit so it's not confusing for students.

**Comment by lizbur10** [comment]
OK I obviously didn't read the instructions very carefully. I think that's fine. My only suggestion would be to use a more descriptive component name like ReusableCard or something. (Hopefully you can think of something better than that)

**Comment by Matt-Eva** [comment]
Good idea! I also just pushed up two changes to update those tests. The variables are just declared directly within the files now, rather than being imported from data.js

**Comment by Matt-Eva** [comment]
Ok, updated README to include more explicit hints and renamed Card component to Reusable Card!

**Pull request merged**